### PR TITLE
Update Contributing guide regarding CI skip

### PR DESCRIPTION
**Pull request by rpsfonseca**
### Description

This pull request is to propose changes in the Contributing guide to make it less prone to errors, as discussed in [PR 10848](https://github.com/astropy/astropy/pull/10848)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -113,11 +113,8 @@ Other Tips
 - To skip only the tests running on Travis CI use ``[skip travis]``.
   This will still execute CircleCI.
 
-- If your commit makes substantial changes to the documentation but no code
-  changes, then you can use ``[ci skip]``, which will skip all CI except RTD,
-  where the documentation is built. The exception to this rule
-  is when your changes to documentation include code snippets that need to
-  be tested using ``doctest``.
+- If your commit makes substantial changes to the documentation but none of those changes include code snippets, then you can use ``[ci skip]``, which will skip all CI except RTD,
```
Comment (pllim) [inline CONTRIBUTING.md:116]: Can you please break this long line into multiple shorter lines to be more consistent with the rest of the document? Thanks!

Comment (rpsfonseca) [reply CONTRIBUTING.md]: Done!

**Review by pllim — APPROVED** [review]
The failures are unrelated and will be investigate separately if they persist on `master` branch.

Thank you!

**Comment by rpsfonseca** [comment]
Thank you! 🙏

**Pull request merged**